src/lib/style/components/_governance-dashboard.scss

```diff
@@ -35,12 +35,18 @@
   }
 }
 
-.AccessRequirementDashboard {
+.AccessRequirementDashboard,
+.AccessSubmissionDashboard {
   .InputPanel {
     display: grid;
     grid-template-columns: auto 265px auto;
     gap: 25px;
 
+    .form-control {
+      border: none;
+      background-color: #ebebeb;
+    }
+
```

Comment:
Let's leave the general styling of the inputs alone. If possible, we should make the change in a place that generalizes across the platform (out of scope for this PR).
```suggestion
```